src/main/java/core/basesyntax/BuilderTestApp.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
 public class BuilderTestApp {
-  public static void main(String[] args) {
-      // Remove this comment and test your Builder implementation here...
-  }
+    public static void main(String[] args) {
+        Plane.PlaneBuilder planeBuilder = new Plane.PlaneBuilder()
+                .setModel("T25")
+                .setColor("White");
+
+        Plane planeYear = planeBuilder.setYearIssue(19990).build();
+        System.out.println(planeYear.toString());
```

Comment:
```suggestion
        System.out.println(planeYear);
```
you can write like this as well - let's be consistent

Reply:
done